politeiawww/proposals.go

```diff
@@ -3336,3 +3336,18 @@ func (p *politeiawww) processVoteDetailsV2(token string) (*www2.VoteDetailsReply
 
 	return vdr, nil
 }
+
+// initLoadVoteResults is used to send the LoadVoteResults decred plugin command
+// to the cache on www startup.
+func (p *politeiawww) initLoadVoteResults() error {
```

Comment:
`initLoadVoteResults` doesn't really make sense.  You're initializing the vote results by sending the `LoadVoteResults` command.  Just `initVoteResults` is better.